pkg/auth/authenticator/password/ldappassword/ldap.go

```diff
@@ -164,10 +126,10 @@ func (a *Authenticator) getIdentity(username, password string) (authapi.UserIden
 
 	// Build list of attributes to retrieve
 	attrs := util.NewStringSet(a.options.URL.QueryAttribute)
-	attrs.Insert(a.options.AttributeEmail...)
-	attrs.Insert(a.options.AttributeName...)
-	attrs.Insert(a.options.AttributePreferredUsername...)
-	attrs.Insert(a.options.AttributeID...)
+	attrs.Insert(a.options.IdentityAttributeMapping.EmailAttribute...)
```

Comment:
Is it more reasonable to have a `RequestedAttributes` method on `LDAPUserToIdentityMapping` that returns a `util.StringSet` of these?